.github/workflows/tests.yml

```diff
@@ -43,9 +47,17 @@ jobs:
           composer --version
           composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
           composer require "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update --dev
+          composer require "larastan/larastan:${{ matrix.larastan }}" --no-interaction --no-update --dev
+          composer require "laravel/pint:${{ matrix.pint }}" --no-interaction --no-update --dev
           composer update --prefer-dist --no-interaction --no-suggest --dev
           composer dump
 
+      - name: Run Pint
+        run: composer check-style
+
+      - name: Run Larastan
+        run: composer static-analysis
```

Comment:
How bad it would be to add this into a different "job"? This way, each one of those shows up in the "checks" section as a standalone item

Reply:
Not bad, just updated it to do it this way with linting and static analysis happening first for the newest version of PHP and Laravel. If one of those fail ([for example](https://github.com/kirschbaum-development/commentions/actions/runs/14887952359)) then it skips running the matrix of tests.